w3-account.md

```diff
@@ -12,61 +12,93 @@
 
 # Abstract
 
-The w3 family of open protocols defines user interactions with self-certified [PKI] based namespaces. These namespaces can be accessed through delegated [UCAN] capabilities that must be synchronized across multiple user agents on multiple devices. Since this is an open and decentralized system, there are unique challenges in providing a user-friendly experience, which is addressed through the concept of an **account** as described in this specification.
+The W3 protocol governs user interactions within self-certified Public Key Infrastructure (PKI)-based namespaces. Access control to these namespaces, for simplicity referred to as spaces, is managed through delegated capabilities in [UCAN] format.
+
+Users access their spaces across various user agents operating on multiple devices. Here we introduce [account] primitive designed to enable synchronization of the access across authorized user agents with a user-friendly experience.
 
 ## Language
 
 The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in [RFC2119](https://datatracker.ietf.org/doc/html/rfc2119).
 
 # Introduction
 
-In w3 family of protocols, a namespace is identified by a [`did:key`] identifier, which means that the owner of that namespace _(private key holder)_ has absolute authority over it. They can delegate some or all of the capabilities for the resources under that namespace to any other [principal]. However, managing these delegations across multiple user agents on multiple devices presents several user experience challenges:
+In W3 protocol, a namespace, or space for short, corresponds to an asymmetric keypair and is identified by a [`did:key`] URI. The private key holder, is the [owner] of that namespace and has absolute authority over it. They can delegate limited or absolute authority over the namespace to any other [principal]. However, managing delegations across multiple user agents on multiple devices presents several user experience challenges:
+
+1. To synchronizing namespace access across user agents they need to discover their [`did:key`] identifiers and level of access granted to them.
+2. Recovering access in case when all devices are lost becomes impossible.
+
+To address these issues, we propose the concept of an account. Account SHOULD have a human-meaningful identifier such as email address. We propose use of email addresses as account identifiers so that derived [`did:mailto`] can act as the [principal] in the [UCAN] delegation chain. This creates [principal] that can be used to aggregate capabilities and manage them.
```

Comment:
> Account SHOULD have a human-meaningful identifier such as email address.

just curious, but why add this recommendation? is MAY sufficient? is there anything bad about a `did:key` account?

Reply:
There is nothing wrong with aggregating capabilities under the `did:key` principal, but I think calling it an account is just misleading. It just semantics, but I think they matter. For example you can not login as `did:key` one because you can't expect user to type it in, but even they could we simply do not have flows to support that, if you'll try to request authorization from `did:key` we will error.

So I think we should not call it account unless you can login with it.